<launch>
  <node name="esp32_camera_node" pkg="stereo_depth_pkg" type="esp32_camera_node.py" output="screen">
    <param name="left_camera_url" value="http://192.168.1.121" />
    <param name="right_camera_url" value="http://192.168.1.129" />
  </node>
  
  <node name="depth_map_node" pkg="stereo_depth_pkg" type="depth_map_node.py" output="screen" />
</launch>
